### DISCUSSION TITLE
Ideas How to Reduce VAE VRAM Usage when saving? (Decode VAE OOM with Hypertiles)

### DISCUSSION THREAD

**Discussion by YourFriendlyNeighborhoodMONKE** [vladmandic/sdnext#2678] [Q&A]
I'm experimenting pushing the envelope with file sizes than I'm normally restricted to by utilizing Unet Hypertiling

However, VAE decoding becomes a bottleneck after the image has been generated and it will just dump RGBA image
`Saving RGBA image as JPEG: Alpha channel will be lost`

Are there any hardcore VAE memory reduction tips I could try to reduce VAE memory usage?

**Comment by vladmandic** [comment]
- use tasesd vae instead of full (set full quality to off)
- enable vae slicing and tiling
- enable move model

**Comment by YourFriendlyNeighborhoodMONKE** [comment]
Thanks for the fast response!

Setting full quality off works, but the results are subpar unsurprisingly

I already had slicing and tiling on

I'm not entirely sure what you mean by move model exactly, but it sounds like something that would be the solution
Could you plase be more specific what I need to do there, please?

PS- I tried using VAE Hypertiling by making the settings visible from the code (they were set False by default), but every resolution number I tried resulted in division errors - Auto's current repo seems to have those settings visible, but couldn't get it working either

**Reply by vladmandic** [reply]
VAE hypertiling is enabled in dev branch if you want to experiment.

Move model options are in settings - diffusers.
